resources/lib/oojs-ui/oojs-ui.js

```diff
@@ -2728,6 +2728,9 @@ OO.ui.WindowManager.static.sizes = {
 	},
 	'800px': {
 		width: 800
+	},
+	page: {
```

Comment:
Or is it not worth it? I'd either modify the VE core or the OO UI one... @Grunny ?